### ISSUE TITLE
Indicate Which Colors are Default on Color Scheme

### ISSUE THREAD

**Issue by NorthernRealities** [Flowwie/flowwie-freecad#14]
As I was using the "Flowwie's secret FreeCAD 0.19 color scheme" PDF I came across a handful of colours that were not changed from the default values. It would be a nice addition to indicate those values that one does not have to change to save the user from having to compare the numbers. I appreciate the effort you went through to make the document in the first place and I thank you very much for it (and the tutorials). I'm just suggesting a small enhancement and not intending to be critical.

**Comment by Flowwie** [comment]
No, it's a great suggestion for the future. in FreeCAD 0.20 I will finally be able to share my color scheme to be downloaded and applied automatically without doing all settings manually. best regards

**Comment by fkuep** [comment]
> in FreeCAD 0.20 I will finally be able to share my color scheme to be downloaded

- Is that available ? 
- Where can I download ?
Thank You :heart:

**Comment by Nailuj1** [comment]
> No, it's a great suggestion for the future. in FreeCAD 0.20 I will finally be able to share my color scheme to be downloaded and applied automatically without doing all settings manually. best regards

I'm also interested ;)

**Comment by arcDaniel84** [comment]
> No, it's a great suggestion for the future. in FreeCAD 0.20 I will finally be able to share my color scheme to be downloaded and applied automatically without doing all settings manually. best regards

Nice, I like you colors, but in 0.20.2 I'm a little confused with you pdf, so I would like an "new" pdf or the color profile to download, that would be very nice.

Thx for you work

**Comment by Kagukara** [comment]
I just came across the video to get into FreeCAD, and saw that the custom colour scheme didn't match the pdf so I converted the names. Its ordered the same way as the pdf.

[FreeCAD-0.19-0.20.2-color-scheme-name-conversion.txt](https://github.com/Flowwie/flowwie-freecad/files/11027333/FreeCAD-0.19-0.20.2-color-scheme-name-conversion.txt)
